packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -206,13 +210,15 @@ class ImagePicker {
     double? maxHeight,
     int? imageQuality,
     CameraDevice preferredCameraDevice = CameraDevice.rear,
+    bool requestFullMetadata = true,
```

Comment:
The default should be false, it makes more sense even if it is a breaking change for some.

Reply:
> Imo, the default should be false, it makes more sense

If this were a new API, I would agree it should default to false, but it's not. We wouldn't make a breaking change to the plugin just for this.

Feel free to file an issue to track changing the default the next time we decide to make a breaking change for other reasons.

> even if it is a breaking change for some.

There's no such thing as a breaking change "for some" in a plugin. A breaking change will change the major version, which affects every client of the plugin whether they would be broken by the specific change or not.